### PR TITLE
AliceVision.Meshroom version 2023.3.0

### PR DESCRIPTION
**Pull request by chausner**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/153041)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\Meshroom.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND

(Automated response - build 880.)

File: manifests/a/AliceVision/Meshroom/2023.3.0/AliceVision.Meshroom.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: AliceVision.Meshroom
+PackageVersion: 2023.3.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Meshroom-2023.3.0\Meshroom.exe
+Installers:
```
Comment (stephengillie) [inline manifests/a/AliceVision/Meshroom/2023.3.0/AliceVision.Meshroom.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by chausner** [comment]
Closing this since the installed portable application does not run when executed from the command-line. It cannot load Python37.dll which is included in the application folder but when the application is run via the command-line from a different working directory, Windows will not find the DLL.

https://github.com/microsoft/winget-cli/issues/2299 or a similar mechanism should provide a solution in the future.

**Comment by stephengillie** [comment]
I started this late on Friday, and have returned to find this message: 
image

**Pull request closed without merging**